README.md

```diff
@@ -41,23 +41,16 @@ Rune provides a challenge number through `Rune.getChallengeNumber()` that will i
 - Changes your game content every day so to ensure your game is always fresh and entertaining
 - Makes everyone compete under the same conditions to ensure fairness and increase competition
 
-You could e.g. use the challenge number in the following ways:
+Here's two common ways you could use the challenge number:
 
 <details>
-<summary>1) As a seed for randomness</summary>
-&nbsp;
-  
-If your game uses randomness to determine the gameplay (e.g. by randomly generating the maps), you can use the challenge number as the seed to achieve deterministic randomness. For example:
+<summary>1) For deterministic randomness (e.g. map generation)</summary>
+&nbsp; 
 
-```js
-import seedrandom from 'seedrandom' // From https://www.npmjs.com/package/seedrandom
+If your game uses randomness to randomly generate maps, you should use `Rune.deterministicRandom()` instead of `Math.random()`. The random number generator provided by Rune is seeded with the challenge number and will therefore always provide the same random values. In this way, all players will play the same map every time.
 
-const challengeNumber = Rune.getChallengeNumber() // Get today's challenge number
-const random = seedrandom(challengeNumber) // Make deterministic number generator
+You should only use `Rune.deterministicRandom()` for your map generation and not as a generic replacement for `Math.random()`. This is because each call to `Rune.deterministicRandom()` will iterate through the random values.
```

Comment:
Perhaps first we could explain how `Rune.deterministicRandom()` works; that it behaves similar to JS function generator, i.e. every time it is called it fetches the next value of an infinite predefined/fixed sequence of `[0, 1)` numbers:
* the sequence is reset on every game over
* the sequences changes every day based on challengeNumber

And then explain how it can be used for generating elements of game maps with example that @mattbretl has provided.

And then why it is important to not replace all `Math.random()` with `Rune.deterministicRandom()`. A concrete example would be great here, e.g. user-interaction animation.

Also, it is not really clear why a game developer would want to call `Rune.deterministicRandom()` twice so maybe we could provide an example, too.

Reply:
Great points. I've incorporated some of this in e.g. https://github.com/rune/rune-games-sdk/pull/39/commits/a23b690e7ba5372412602c2502c288632e46aeec and https://github.com/rune/rune-games-sdk/pull/39/commits/8ce2c14a31fe31b043f2b201fd5766a629dbdcb2.

I've tried to keep it short and simple so haven't included all these points. Similarly, I used a more simplified description than the `[0, 1)` notation in case whoever using the SDK does not have a mathematical background.

Reply:
See also related comment on https://github.com/rune/rune-games-sdk/pull/39#discussion_r819635387.